data/prepro_utils.lua

```diff
@@ -70,6 +70,8 @@ function extractFeatures(model, opt, ndims, preprocessFn)
     if opt.trainSplit == 'train' then
         imFeats = extractFeaturesSplit(model, opt, ndims, preprocessFn, 'val')
         h5File:write('/images_val', imFeats)
+        imFeats = extractFeaturesSplit(model, opt, ndims, preprocessFn, 'test')
+        h5File:write('/images_test', imFeats)
```

Comment:
This will fail for 0.9, right?

Reply:
no it should not. The prepro script populates `unique_img_test` from `v1.0` test for `v0.9`.